app/src/main/java/fr/free/nrw/commons/media/ZoomableActivity.kt

```diff
@@ -650,4 +658,11 @@ class ZoomableActivity : BaseActivity() {
         scope.cancel()
         super.onDestroy()
     }
+
+    companion object {
+        /**
+         * Key for Accessing Intent Data Named "Origin"
```

Comment:
Please explain more: `The value indicates what fragment ZoomableActivity was created by. It is null if ZoomableActivity was created by the custom picker.`